include/SFML/Audio/Music.hpp

```diff
@@ -334,4 +334,4 @@ class SFML_AUDIO_API Music : public SoundStream
 ///
 /// \see sf::Sound, sf::SoundStream
 ///
-////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////
```

Comment:
Why this change?

Reply:
I was fixing some stuff. It's been fixed now. 
I apologies for any inconvenience.